playbooks/hdd_monitoring_writing.yaml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Prepare controller variables
+  import_playbook: prepare_variables.yaml
+  vars:
+    scenario_name: csm_controller hdd_monitoring
+
+- name: Prepare variables
+  hosts: localhost
+  vars_files:
+    - ./vars/csm_controller.yaml
+    - ./vars/hdd_monitoring.yaml
+  tasks:
+    - name: Register nodes
+      add_host:
+        name: "{{ item }}"
+        groups: hdd_hosts
+        ansible_ssh_user: linux
+        ansible_ssh_private_key_file: "{{ csm_controller.controller_key }}"
+        ansible_ssh_common_args: >
+          -o UserKnownHostsFile=/dev/null
+          -o ProxyCommand='ssh -W %h:%p
+          -q linux@{{ csm_controller.csm_controller_fip }}
+          -i {{ csm_controller.controller_key }}
+          -o StrictHostKeyChecking=no
+          -o UserKnownHostsFile=/dev/null'
+          -o PasswordAuthentication=no
+      loop: "{{ hdd_monitoring.hdd_ecs_local_ips }}"
+
+- name: Start test
+  hosts: hdd_hosts
+  become: yes
+  vars:
+    devices:
+      sata: /dev/vdb
+      sas: /dev/vdc
+      ssd: /dev/vdd
+  tasks:
+    - name: Create big files on all devices every 5 minutes
```

Comment:
It's not running every 5 minutes :)

Reply:
this part to remember when we include this in apimon)